mne/preprocessing/ica.py

```diff
@@ -2084,3 +2101,230 @@ def _band_pass_filter(ica, sources, target, l_freq, h_freq, verbose=None):
         raise ValueError('Must specify both pass bands')
 
     return sources, target
+
+
+@verbose
+def corrmap(icas, template, threshold="auto", label="bads",
+            plot=True, copy=False, ch_type="eeg"):
```

Comment:
I find it maybe a bit implicit but consistency with the orginal implementation is not a bad argument. If it simplfies our API it's good.